.github/workflows/ci.yml

```diff
@@ -14,17 +15,12 @@ jobs:
 
     runs-on: ubuntu-latest
 
-    strategy:
-      matrix:
-        node-version: [16.x, 18.x] # 19.x stalling on seattlecolleges repo but running fine on my forked repo..
-        # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
-
     steps:
       - uses: actions/checkout@v3
-      - name: Use Node.js ${{ matrix.node-version }}
+      - name: Use Node.js 16
```

Comment:
Current LTS version is 18, so let's bump the nodejs version from 16 to 18. 
<img width="659" alt="Screenshot 2023-08-31 at 2 44 23 PM" src="https://github.com/SeattleColleges/belindas-closet-nextjs/assets/6230110/67fb220b-c0a1-4262-a27e-9b6f54cf360b">